pkg/rpc/server/subscription.go

```diff
@@ -58,12 +58,12 @@ func (f *feed) Matches(r *response.Notification) bool {
 	case response.TransactionEventID:
 		filt := f.filter.(request.TxFilter)
 		tx := r.Payload[0].(*transaction.Transaction)
-		senderOK := filt.Sender == nil || tx.Sender.Equals(*filt.Sender)
+		senderOK := filt.Sender == nil || tx.Sender().Equals(*filt.Sender)
 		cosignerOK := true
 		if filt.Cosigner != nil {
```

Comment:
It should be `filt.Signer` now with the respective `range tx.Signers` semantics. Client and documentation are to be updated too.

Reply:
Did you mean that we should remove `filt.Sender` or just rename `filt.Cosigner` and allow to use the sender as `filt.Signer`?
Don't we want to distinguish sender and cosigners in subscriptions? I thought that it might be useful, because we don't have scopes in subscriptions and there's no need to subscribe to sender in filt.Signer when you can just subscribe to filt.Sender.

Reply:
> rename filt.Cosigner and allow to use the sender as filt.Signer

Like this one.

> Don't we want to distinguish sender and cosigners in subscriptions?

Well, I might be wrong but I see two main use cases --- either you care about some address being a sender or you care about some address being one of signers (including being sender). Filtering for signer address which is not a sender seems a bit suspicious. Maybe there is a use case for it too, but I can't see any at the moment.